src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import App from './app/App'
```

Comment:
you don't use export correct 
```suggestion
import { App } from './app/App';
```

Reply:
fixed